src/library_noderawfs.js

```diff
@@ -55,6 +54,12 @@ addToLibrary({
       var mode = NODEFS.getMode(path);
       return { path, node: { id: st.ino, mode, node_ops: NODERAWFS, path }};
     },
+    createStandardStreams() {
+      FS.createStream({ nfd: 0, position: 0, path: '', flags: 0, tty: false, seekable: false }, 0);
```

Comment:
Note that I set `tty: false` here to ensure that issue #22264 remains resolved.
```console
$ emcc ioctl_test.c -lnodefs.js -lnoderawfs.js -sEXIT_RUNTIME
$ node ./a.out.js
pc-kaw
a.out.js: /dev/stdin not a tty: Not a tty
```

Reply:
(I also set it for stdout and stderr, btw)

Reply:
Seems reasonable for now, but we should probably add a TODO there, and we should really be setting `tty` based on the underlying node `isatty` call for each descriptor.

Reply:
Ah, I just noticed that `isatty` for stdio streams will return `0` and set `errno` to `ENOTTY` with this change, which is probably not what we want. I just pushed commit 88a5475f9e68be66d24d77ed974ecc65fbfa6947 instead. Let's reopen #22264 after this change.